### DISCUSSION TITLE
[✨] deprecate signals on regular DOM attributes

### DISCUSSION THREAD

**Discussion by wmertens** [QwikDev/qwik-evolution#137] [Proposals For Qwik]
### Is your feature request related to a problem?

When writing a component that forwards class, the correct way to forward it is `(props.class as Signal<string>)?.value ?? props.class as string`.

This is a mouthful and there's no reason to support this, because the optimizer will forward class={signal.value} correctly.

### Describe the solution you'd like

Deprecate signals as DOM attribute values except where expected, like bind: value

### Describe alternatives you've considered

.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
